java-allocation-instrumenter/pom.xml

```diff
@@ -223,4 +223,36 @@
       <organization>Google Inc.</organization>
     </developer>
   </developers>
+  
+  <profiles>
+    <profile>
+      <id>jdk8</id>
+      <activation>
+        <jdk>[1.8,)</jdk>
+      </activation>
+      <!-- Disable doclint under JDK 8 -->
```

Comment:
I think doc linting was made stricter or something in JDK8. For example, `mvn clean install` using JDK8 for this project fails with a bunch of errors like this:

```
[ERROR] public static byte[] instrument(byte[] originalBytes, String recorderClass,
[ERROR] ^
[ERROR] /usr/local/google/home/cgdecker/Projects/allocation-instrumenter/java-allocation-instrumenter/src/main/java/com/google/monitoring/runtime/instrumentation/AllocationInstrumenter.java:207: warning: no @param for originalBytes
...
```

There's something the same or similar in the `pom.xml` for Guice, Guava, Jimfs etc. already.